[2023-02-01T00:00:02.625+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-31T12:00:00+00:00 [queued]>
[2023-02-01T00:00:02.633+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-31T12:00:00+00:00 [queued]>
[2023-02-01T00:00:02.633+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-02-01T00:00:02.633+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-02-01T00:00:02.633+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-02-01T00:00:02.643+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_avaible> on 2023-01-31 12:00:00+00:00
[2023-02-01T00:00:02.648+0000] {standard_task_runner.py:55} INFO - Started process 1196 to run task
[2023-02-01T00:00:02.650+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_avaible', 'scheduled__2023-01-31T12:00:00+00:00', '--job-id', '420', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmprk1ukc8t']
[2023-02-01T00:00:02.651+0000] {standard_task_runner.py:83} INFO - Job 420: Subtask is_api_avaible
[2023-02-01T00:00:02.692+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-31T12:00:00+00:00 [running]> on host 35d887eb39bb
[2023-02-01T00:00:02.738+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_avaible
AIRFLOW_CTX_EXECUTION_DATE=2023-01-31T12:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-31T12:00:00+00:00
[2023-02-01T00:00:02.739+0000] {http.py:120} INFO - Poking: api/
[2023-02-01T00:00:02.744+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-02-01T00:00:02.745+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-02-01T00:00:03.409+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-02-01T00:00:03.415+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_avaible, execution_date=20230131T120000, start_date=20230201T000002, end_date=20230201T000003
[2023-02-01T00:00:03.464+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-02-01T00:00:03.491+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
